### PR TITLE
Port window_next_sibling and window_prev_sibling

### PR DESCRIPTION
**Pull request by mehakun**

### CODE REVIEW

File: rust_src/src/windows.rs [hunk 1]
```diff
@@ -1252,6 +1252,24 @@ pub fn window_next_buffers(window: LispWindowLiveOrSelected) -> LispObject {
     win.next_buffers
 }
```
Comment (shaleh) [inline rust_src/src/windows.rs]: Close. Both lisp_fns need `min = "0"` because the DEFUN says "0, 1, 0" which means min 0, max 1, not interactive.

Comment (mehakun) [reply rust_src/src/windows.rs]: oh, yeah, thx. Read porting guide poorly

Comment (shaleh) [reply rust_src/src/windows.rs]: In rust_src/admin there is a script called rustize-emacs. C-u M-| is a good way to call it.

Comment (mehakun) [reply rust_src/src/windows.rs]: added `min`'s

**Pull request merged**